app/views/idv/doc_auth/verify.html.slim

```diff
@@ -0,0 +1,24 @@
+-title t('doc_auth.titles.doc_auth')
+
+h1.h3 = t('doc_auth.headings.verify')
+
+div = "#{t('doc_auth.forms.first_name')}: #{flow_session[:pii_from_doc][:first_name]}"
+div = "#{t('doc_auth.forms.last_name')}: #{flow_session[:pii_from_doc][:last_name]}"
+div = "#{t('doc_auth.forms.dob')}: #{flow_session[:pii_from_doc][:dob]}"
+hr
+.right = link_to(t('doc_auth.buttons.change_address'), idv_address_url)
+div = "#{t('doc_auth.forms.address1')}: #{flow_session[:pii_from_doc][:address1]}"
+div = "#{t('doc_auth.forms.city')}: #{flow_session[:pii_from_doc][:city]}"
+div = "#{t('doc_auth.forms.state')}: #{flow_session[:pii_from_doc][:state]}"
+div = "#{t('doc_auth.forms.zip_code')}: #{flow_session[:pii_from_doc][:zipcode]}"
+hr
+= form_for('change ssn', url: idv_doc_auth_step_path(:redo_ssn), method: 'PUT') do
```

Comment:
Why is this `button_to` helper inside a form? `button_to` will create a form for you. It looks like the outside form is unused.

Reply:
Originally I had a page form and the button_to was refusing to create a nested form so I created one manually.  Later I moved the page form out of there when I realized it was only wrapping the continue button.  Rather than circling back and converting them both to not use forms I just left them and here we are :-) They are now gone...poof!